tests/e2e/pact/broker/publish.ts

```diff
@@ -1,8 +1,8 @@
 import path from "path"
 import { Publisher } from "@pact-foundation/pact"
 
-async function publish() { 
-  await new Publisher({
+async function publish(): Promise<string[]> { 
```

Comment:
not sure the linter is up to date in this repo (which means it may need updating) but think we went with `Array<string>` over `string[]`